File: DataStructures/com/arrays/IsArraySorted.java
package com.arrays;

public class IsArraySorted {

    public static boolean isArraySorted(int[] arr, int n){

        for(int i = 0; i < n-1; i++){
            if( arr[i+1] < arr[i])
                return false;
        }
        return true;
    }

    public static void main(String[] args) {
        int[] array = new int[]{1, 2, 2, 3, 3, 4};
        int n = array.length;

        System.out.println(isArraySorted(array, n));
    }
}

File: Tutorial/JSProblems/isArraySorted.js


function IsSorted(arr){

  if( arr.length === 1)
  return true;
 // console.log(arr.length);

  for( let i = 0; i <= arr.length - 1; i++){
      if(arr[i] > arr[i+1])
      return false;
  }
  return true;
}

// console.log(IsSorted([1, 2, 3]));
export default IsSorted;